add-on/chrome/test/mochitest/browser_LoopSidebar.js

```diff
@@ -0,0 +1,65 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/*
+ * This file contains tests for the LoopSidebar
+ */
+"use strict";
+
+const { LoopSidebar } = Cu.import("chrome://loop/content/modules/LoopSidebar.jsm", {});
+
+var win;
+
+registerCleanupFunction(function* () {
+  if (win) {
+    yield BrowserTestUtils.closeWindow(win);
+  }
+});
+
+function* closeWindow() {
+  if (!win) {
+    win = window;
```

Comment:
We shouldn't need the if() here, nor the assignment, if its null, there's an error.